cmds/more/more.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// More pages through files without any terminal trickery.
+//
+// Synopsis:
+//     more [OPTIONS] FILE
+//
+// Options:
+//     --lines NUMBER: screen size in number of lines
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+
+	flag "github.com/spf13/pflag"
+)
+
+var lines = flag.Int("lines", 40, "screen size in number of lines")
+
+func main() {
+	flag.Parse()
+	if flag.NArg() != 1 {
+		log.Fatal("more can only take one file")
+	}
+	if *lines <= 0 {
+		log.Fatal("lines must be positive")
+	}
+
+	f, err := os.Open(flag.Args()[0])
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	scanner := bufio.NewScanner(f)
+	for i := 0; scanner.Scan(); i++ {
+		if (i+1)%*lines == 0 {
+			fmt.Print(scanner.Text())
```

Comment:
one of the most annoying things about more is that it will block on, e.g., a named pipe until it gets a screenful. Is there some way to have this output lines until it hits the limit, rather than not outputting lines until it hits the limit?

Reply:
Right now, it prints them out line-by-line. If a line is very long, or is taking time for Read to return, it won't be printed. Is this okay?

Also, there is a hidden unfortunate things about `bufio.Scanner`. When a line is longer than 64KiB, it errors:

    2018/09/25 08:09:30 bufio.Scanner: token too long

Reply:
can you get rid of bufio then ... that's a stupid error?